.github/workflows/rust.yml

```diff
@@ -16,11 +16,14 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+    - uses: actions-rs/toolchain@v1
```

Comment:
https://github.com/banyancomputer/ipfs-proof-buddy/blob/main/.github/workflows/ci.yml
https://github.com/banyancomputer/.github/blob/master/.github/workflows/rust-codecov.yml

Reply:
@amiller68 ^^

Reply:
rebasing off master will resolve this